packages/kbn-ts-projects/ts_project.ts

```diff
@@ -36,14 +36,15 @@ function isValidKbnRefs(refs: unknown): refs is KbnRef[] {
 }
 
 function expand(name: string, patterns: string[], knownPaths: string[]) {
+  const pathsArray = Array.isArray(knownPaths) ? knownPaths : [];
   const matchers = patterns.map((pattern) => ({
     pattern,
     matcher: makeMatcher([pattern]),
   }));
 
   const selected = matchers.map(({ matcher, pattern }) => ({
     pattern,
-    matches: knownPaths.filter(matcher),
+    matches: pathsArray.filter(matcher),
```

Comment:
was this a bug?

Reply:
My Linter complained about it - ".filter is not a function" but I think it should never actually hit this sort of error since the signature of the function specifies the type of knownPaths. 

Popped a check in to appease the linter, but wanted to know what others think for this.

Reply:
This shouldn't have thrown a linter error. My guess is you have something else configured in your IDE which is throwing the error.

Reply:
Thanks for the confirmation @Ikuni17 I will revert this part of the change then

Reply:
@CAWilson94 feel free to reach out to me or the Operations team if you need help fixing your environment.